aws/ec2/listImages/package.json

```diff
@@ -9,8 +9,10 @@
   "author": "Cal Supik",
   "license": "MIT",
   "dependencies": {
+    "@risk3sixty/extension-sdk": "^0.0.8",
     "aws-sdk": "^2.795.0",
-    "columnify": "^1.5.4"
+    "columnify": "^1.5.4",
+    "moment": "^2.29.1"
```

Comment:
While this is more of a nitpick than anything, I recommend looking into `dayjs` for small projects/utilities for time handling. It's got an identical API as moment but is way more lightweight. It strips out a lot of the locale stuff and what not that moment ships with and its like an order of magnitude or two smaller.

Reply:
7c592c3